### ISSUE TITLE
Auto Loot holotape doesn't appear if you add it mid save file

### ISSUE THREAD

**Issue by Skyshayde** [fireundubh/fallout4#24]
I just installed Auto Loot and I haven't been given the holotape.  I can confirm that it is installed (I can see the quest in the console) and I have tried force triggering the quest which gives the holotape to give it to me, but it didn't work.  As near as I can tell, it gives you the holotape sometime during one of the early main quests (Which is kinda weird.  Am I misreading the papyrus scripts?)  Since I am after this quest, it never triggers.

**Comment by fireundubh** [comment]
You're misreading the script.

The holotape is added to your inventory when you first pick up the Pip-Boy or any time after then.

Stage 6 of MQ102 is before you pick up the Pip-Boy. When you pick up the Pip-Boy, Stage 6 is completed.

`MQ102.IsStageDone(6)` returns true if Stage 6 was completed (i.e., you picked up the Pip-Boy), and if true, the quest is shut down and the Pip-Boy is added to your inventory. If `IsStageDone()` returns false, the script continues to run and do nothing until that function returns true.

The `OnQuestShutdown()` event might be as unreliable as the `OnEffectFinish()` event, however. Yet, I have never experienced the issue of the holotape not being added, and I always load an existing unmodded save to skip Vault 111.

In the current version, the quest runs only once, so if the quest was shut down and never added the holotape for whatever reason, and you saved, it will not run again when you load the same save. The next version uses a similar script from Auto Lockpick, and the holotape quest is no longer flagged as Run Once, so that the quest always starts enabled no matter what.

The drawback here is that you can't get rid of the holotape, and trying to get rid of the holotape will only trigger the script to add another one. Hmm, I should flag the holotape as a quest item.

```
Event OnQuestInit()
	StartTimer(5)
EndEvent

Event OnTimer(Int aiTimerID)
	If !MQ102.IsStageDone(6)
		StartTimer(5)
	Else
		If Player.GetItemCount(dubhAutoLootHolotape) == 0
			Player.AddItem(dubhAutoLootHolotape, 1, False)
			Self.Stop()
		Else
			Self.Stop()
		EndIf
	EndIf
EndEvent
```

**Comment by Skyshayde** [comment]
Well, I've figured out what caused the problem.  I'm running an alternate start mod which skips that part of the quest.  Running setstage mq102 6 on a new save solves the problem, although that doesn't work once the quest is completed.  I'll update this if I find a way to change that.

**Comment by Skyshayde** [comment]
I managed to fix it by adding the item "440017AC".

**Comment by fireundubh** [comment]
Best to report that to the creator of the alternate start mod. That mod should be manually completing all quests and quest stages that need to be completed. Otherwise, my mod isn't the only thing that'll break. Vanilla quests and scripts will break, too.
